testcases/contenttype.http

```diff
@@ -0,0 +1,11 @@
+POST / HTTP/1.1
+Host: handler.example.com
+Content-Type: application/xml
+Content-Length: 24
+CE-specversion: 0.3
```

Comment:
s/CE-/ce-/g
s/0.3/0.4-wip/g

Reply:
HTTP is case-insensitive, So-RaNSoMcAsE-iS-fInE.

Updated anyway. 😁